packages/protocol/contracts/L1/ProverPool.sol

```diff
@@ -93,8 +91,7 @@ contract ProverPool is EssentialContract, IProverPool {
     }
 
     /**
-     * @notice Protocol specifies the current feePerGas and assigns a prover to
-     * a block.
+     * Protocol specifies the current feePerGas and assigns a prover to a block.
```

Comment:
use inheritdoc here as well?

Reply:
inheritdoc is not valid for contracts. i am following the same standard as sablier-v2 here: https://github.com/sablier-labs/v2-core/blob/main/src/SablierV2Comptroller.sol#L30

i added `@dev` to describe implementation specific details to the interface. i will add that as a standard in our documentation spec.